jax/_src/numpy/ufuncs.py

```diff
@@ -3150,9 +3150,36 @@ def hypot(x1: ArrayLike, x2: ArrayLike, /) -> Array:
   return _where(idx_inf, _lax_const(x, np.inf), x)
 
 
-@implements(np.reciprocal, module='numpy')
 @partial(jit, inline=True)
 def reciprocal(x: ArrayLike, /) -> Array:
+  """Calculate element-wise reciprocal of the input.
+
+  JAX implementation of :obj:`numpy.reciprocal`.
+
+  The reciprocal is calculated by ``1/x``.
+
+  Args:
+    x: input array or scalar.
+
+  Returns:
+    An array of same shape as ``x`` containing the reciprocal of each element of
+    ``x``.
+
+  Note:
+    - ``jnp.reciprocal`` returns the reciprocal of the integer inputs, unlike
+      ``numpy.reciprocal`` which returns ``0`` for all the integer inputs.
+    - ``jnp.reciprocal`` returns ``inf`` for input ``0`` unlike ``numpy.reciprocal``
+      which overflows with a ``RuntimeWarning``.
```

Comment:
To me this implies that the return value is different, but `jnp.reciprocal` and `np.reciprocal` both return `inf` for floating point zero inputs. This is just a case of the general property that JAX does not produce warnings in the host due to invalid runtime values, while NumPy does. I might just remove this since it's discussed elsewhere

Reply:
Thanks! Removed.